src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -85,13 +85,13 @@ public function add($child, $type = null, array $options = [])
      *
      * This method should not be invoked.
      *
-     * @param string                   $name
-     * @param string|FormTypeInterface $type
```

Comment:
I removed `FormTypeInterface` because it was not supported by the factory anyway. The test was working with a mock factory and was not fully checking this case.